src/extension.ts

```diff
@@ -7,19 +7,28 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *********************************************************************/
-import { ExtensionContext, commands, window } from 'vscode';
+import { ExtensionContext, commands, window, debug } from 'vscode';
 import { MemoryServer } from './memory/server/MemoryServer';
 export { MemoryServer } from './memory/server/MemoryServer';
 import { ResumeAllSession } from './ResumeAllSession';
 export { ResumeAllSession } from './ResumeAllSession';
 import { SuspendAllSession } from './SuspendAllSession';
 export { SuspendAllSession } from './SuspendAllSession';
 
+
 export function activate(context: ExtensionContext) {
     new MemoryServer(context);
     new ResumeAllSession(context);
     new SuspendAllSession(context);
 
+    debug.onDidReceiveDebugSessionCustomEvent(event => {
+        if (event.event === "UpdateBreakpointView") {
+          const bps = debug.breakpoints;
```

Comment:
You could check if there are any breakpoints to update. In my case I set a `tbreak` and deleted it fro the console. breakpoints were empty/undefined.